webview/src/setup/components/experiments/styles.module.scss

```diff
@@ -8,3 +8,15 @@
     margin: 0 auto;
   }
 }
+
```

Comment:
Minor, but with this change, a couple tabs are wider/shorter than the others causing the content to "jump" in width:

https://github.com/iterative/vscode-dvc/assets/43496356/e7ec0234-f76e-484a-957a-02d2024ea9fd

This can be fixed if we give each tab's content a width of 600px:

https://github.com/iterative/vscode-dvc/assets/43496356/e720b080-901d-42d3-ba0c-a550d9899645

Reply:
We should do the same for all of the panels. I'll follow up.